android/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
correct me if I'm wrong, but `.project` files should be in gitignore

Reply:
same for `perfs` files

Reply:
I found out that `.project` probably should be ignored ([gitignore.io](https://www.gitignore.io/api/reactnative,androidstudio) and [react native's repo](https://github.com/facebook/react-native/blob/master/.gitignore)).

No info about `perfs` files though.